cms/lib/xblock/tagging/tagging.py

```diff
@@ -71,25 +79,36 @@ def save_tags(self, request=None, suffix=None):  # pylint: disable=unused-argume
         """
         Handler to save choosen tags with connected XBlock
         """
-        found = False
-        if 'tag' not in request.params:
-            return Response("The required parameter 'tag' is not passed", status=400)
+        try:
+            posted_data = json.loads(request.body_file.read())
```

Comment:
Oh, strange. That sounds like an issue with the code that's translating from a django request to a webob request. Looking at your stack trace, it looks like you're accessing `request.json` in a print statement. Is it possible that at that point you'd already read the body earlier in the handler method?